vendor.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/Microsoft/go-winio v0.6.2
 	github.com/Microsoft/hcsshim v0.11.4
 	github.com/RackSec/srslog v0.0.0-20180709174129-a4725f04ec91
+	github.com/ThalesIgnite/crypto11 v1.2.5
```

Comment:
This repository also looks dormant; last commit 3 Years ago, which doesn't bring a lot of confidence; https://github.com/ThalesGroup/crypto11

Reply:
I did not notice that, if we decide to go forward with this PR, I will look into finding a better choice here. This package was chosen by a colleague for another place we use PKCS#11 hardware tokens, I'll ask her about it.